Sources/MotorControl/ViewModel/MotionSensorViewModels.swift

```diff
@@ -188,6 +184,18 @@ final class TappingStepViewModel : MotionSensorStepViewModel {
     }
 }
 
+/// View model for a tremor step
+final class TremorStepViewModel : MotionSensorStepViewModel {
+}
+
+/// View model for a walk step
+final class WalkStepViewModel: MotionSensorStepViewModel {
+}
+
+/// View model for a balance step
+final class BalanceStepViewModel: MotionSensorStepViewModel {
```

Comment:
What's the difference between these view models that you need separate classes for them?

Reply:
At least one separate class is needed for Walk or Balance steps because Tremor and Kinetic Tremor use the AppBackgroundListener to quit the measure on interruptions, while Walk or Balance steps do not.

Reply:
Makes sense. Or you could make that config?